src/hotspot/share/utilities/globalDefinitions_gcc.hpp

```diff
@@ -61,6 +72,16 @@
 #include <sys/time.h>
 #endif // LINUX || _ALLBSD_SOURCE
 
+#ifdef _AIX
+#if defined(__open_xl_version__)
+  #if __open_xl_version__ < 17
+  #error "open xlc < 17 not supported"
+  #endif
+#else
+  #error "xlc version not supported, macro __open_xl_version__ not found"
+#endif
+#endif // AIX
```

Comment:
This `#ifdef _AIX` might be obsolete, because configure will throw a warning if the compiler has a lower version, but it's only a warning.

Reply:
I'd prefer having less AIX specific parts in this file. Can this be moved somewhere else? Or maybe combine it with the AIX code above?

Reply:
My question is, do we need this block, because now already configure warns about an outdated compiler, or is a warning to weak and we want to force this error here?

Reply:
I think that building with xlc 16 is no longer possible because the old build pipeline is no longer supported and that is already caught by configure. So, can we even reach here with older xlc compilers?
If not, this code can get removed.

Reply:
Yes, of course you are right. All the compile statements will fail with xlc 16 or older. I will remove it.